react/week2/my-app/src/index.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import Counter from "./counter";
+
+//********************************************************************************************************************************************************* */
+const TodoListItems=({todoitem, ToggleCompleted, DeleteTodo})=>{
+  return <li className={ (todoitem.completed) ? "done": "not-done" }>
+    <div>{todoitem.name}</div>
+    <button onClick={DeleteTodo}>Delete</button>
+    <input type="checkbox" checked={todoitem.completed} onChange={ToggleCompleted}/>
+    <label></label>
+  </li>;
+}
+
+
+
+
+
+
+
+class TodoList extends React.Component {
+  // const TodoList =() => {
+  state ={
+    Todo : [
```

Comment:
You should reserve Uppercased variables for components only: 
```suggestion
    todo : [
```